scraper.rb

```diff
@@ -4,9 +4,10 @@
 require 'scraperwiki'
 require 'nokogiri'
 require 'date'
-require 'open-uri'
+# require 'open-uri'
```

Comment:
NB: I don't think you need to comment this out — it's only `open-uri-cached` ones that will interfere with the archiver — though I don't think it'll do any harm to do so.